ProcessMaker/processmaker
# Add Rebuild Script Executors API Documentation

## Issue & Reproduction Steps
Ticket: [FOUR-5986](https://processmaker.atlassian.net/browse/FOUR-5986)

As a process architect, I want to have documentation of the Rebuild Script Executors API, so I can manage Rebuild Script Executors

## Solution
- Add missing API documentation

## How to Test
1. Navigate to `/api/documentation` in your local instance.
2. Authorize yourself via Bearer Token
3. Locate the 'Signals' location
4. 'Try Out' all APIs
5. Ensure all fields work properly
6. Ensure all properties are listed in the example schema

## Related Tickets & Packages
NONE

## Code Review Checklist
